### DISCUSSION TITLE
Gost Apps menu.

### DISCUSSION THREAD

**Discussion by alphamodh0** [helloSystem/hello#305] [Q&A]
I gor this after remove Thunderbird from Apps folder.
![20220102_103101.jpg](https://user-images.githubusercontent.com/90866187/147880686-46831fdf-2903-4b83-a520-2b5c6c32e926.jpg)

**Comment by probonopd** [comment]
This should not be. Please log out and log in again and see whether this fixes it.

**Comment by alphamodh0** [comment]
![20220102_103726.jpg](https://user-images.githubusercontent.com/90866187/147880863-3194ff5b-337d-49a5-88cd-7c4c647b2e18.jpg) Got same bug 🥺.

**Reply by alphamodh0** [reply]
Also I rebooted.

**Reply by probonopd** [reply]
Now this is very strange.

Please kill the Menu by pressing Ctrl-Alt-Esc and clicking on it, then launch a Terminal by pressing Crl-Alt-T and enter `launch Menu` - what does it say?

**Reply by alphamodh0** [reply]
Menu is realy rare ![20220102_110712.jpg](https://user-images.githubusercontent.com/90866187/147881760-41b19649-51d4-46ac-bbf8-9e58b3d55992.jpg)
![20220102_110643.jpg](https://user-images.githubusercontent.com/90866187/147881785-b5bedb02-42cb-443d-8948-35fc645b5661.jpg)

**Reply by probonopd** [reply]
Something is really messed up badly there. What does `ls /Applications` say?

**Reply by alphamodh0** [reply]
![20220102_123723.jpg](https://user-images.githubusercontent.com/90866187/147884326-afb7636a-eccd-4716-af1d-0b62bdd5c91a.jpg)

**Reply by alphamodh0** [reply]
After a time i decided reinstall the System.

**Reply by probonopd** [reply]
It would be very helpful if we could find a way to recreate the issue, e.g., on the Live ISO (risk free). Can you get the Live ISO to misbehave in the same way?

**Reply by alphamodh0** [reply]
Ok, I'll try it.
